modules/etcd-defrag.adoc

```diff
@@ -6,10 +6,43 @@
 [id="etcd-defrag_{context}"]
 = Defragmenting etcd data
 
-Manual defragmentation must be performed periodically to reclaim disk space after etcd history compaction and other events cause disk fragmentation.
+Defragmentation of etcd data reclaims disk space after etcd history compaction and other events  cause disk fragmentation.
 
 History compaction is performed automatically every five minutes and leaves gaps in the back-end database. This fragmented space is available for use by etcd, but is not available to the host file system. You must defragment etcd to make this space available to the host file system.
 
+Defragmentation occurs using one of the following options:
+* Automatic
+* Manual
+
+[NOTE]
+====
+Automatic defragmentation is preferred because the etcd operator uses cluster information to determine the most efficient operation for the user.
+====
+
+.Automatic defragmentation
+
+Automatic defragmentation is initiated by the etcd operator and occurs during set intervals. This interval is determined by the etcd operator on a time-based basis and no manual intervention is needed.
+
+Verify that the defragmentation process is successful by viewing one of these logs:
+* etcd logs
+* cluster-etcd-operator pod
+* operator status error log
+
+.Example output
+[source,terminal]
+----
+I0907 08:43:12.171919       1
+defragcontroller.go:198] etcd member "ip-
+10-0-191-150.us-west-2.compute.internal"
+backend store fragmented: 39.33 %, dbSize:
+349138944
+----
+
+.Manual Defragmentation
+
+Manual defragmentation must be performed periodically to reclaim disk space after etcd history compaction and other events cause disk fragmentation.
```

Comment:
This makes it sound like automatic defragmentation isn't really functional. Is that the case, or is manual defragging more of a situational tool?